lib/hammer_cli_katello/activation_key.rb

```diff
@@ -234,6 +234,10 @@ class ContentOverrideCommand < HammerCLIKatello::SingleResourceCommand
       desc _("Override product content defaults")
       command_name "content-override"
 
+      validate_options do
+        all(:option_label, :option_value).required
```

Comment:
It's not required on the UI side, they pass in a hash param, while cli passes in these individual values... so if it were set to required, it would fail any UI updates.